src/components/Map/StaticMap.scss

```diff
@@ -0,0 +1,50 @@
+@import "../variables";
+
+.static-map-wrapper {
+  display: flex;
+  // TODO: maybe remove this and let parent element decide the size
+  height: 31.25rem;
+}
+
+.static-map-image {
+  width: 100%;
+  height: 100%;
+  flex: none;
+  z-index: 50;
+
+  img {
+    object-fit: contain;
+  }
+}
+
+.static-map-overlay {
+  width: 100%;
+  margin-left: -100%;
+  z-index: 100;
+  transition: background-color 0.5s;
+  cursor: pointer;
+
+  .button-wrapper {
+    height: 100%;
+    position: relative;
+
+    .button {
+      opacity: 0;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      transform: translate(-50%, -50%);
+      transition: opacity 0.5s;
+    }
+  }
+
+  &:hover {
```

Comment:
Could we make it so that overlay only happens over the map?
![image](https://user-images.githubusercontent.com/33335963/85172275-8d288b00-b225-11ea-8e16-0e654fea4609.png)

Reply:
The image is using `object-fit: contain;` in the css style-sheet to ensure all the markers on the map are visible. But this means that most of the time the image will be smaller than the container and the will be letterbox on the side 😭 

I don' think there is a way to make the overlay happens on top of a image with letterbox, so I updated the file to ensure the map is always the size of the container. The map can also now resize based on the parent container. 

Feels like there is an easier way to do it but 🤷‍♂️